lib/Runtime/Library/TypedArray.cpp

```diff
@@ -509,12 +510,15 @@ namespace Js
             if (args.Info.Count > 3 && !JavascriptOperators::IsUndefinedObject(args[3]))
             {
                 mappedLength = ArrayBuffer::ToIndex(args[3], JSERR_InvalidTypedArrayLength, scriptContext, ArrayBuffer::MaxArrayBufferLength / elementSize, false);
+                newByteLength = mappedLength * elementSize;
 
-                if ((uint32)mappedLength > (byteLength - offset)/ elementSize)
+                if (offset + newByteLength > byteLength)
```

Comment:
No. Please only look at the last commit with message "Resolves Bug#8971506...". The other commits are a result of a merge I did after pushing my branch. But those commits are already merged to release/1.4 branch. Sorry about the confusion.